apps/member-profile/app/routes/_profile.profile.general.tsx

```diff
@@ -174,6 +176,16 @@ export default function UpdateGeneralInformationSection() {
           longitudeName={keys.currentLocationLongitude}
         />
 
+        <InputField
+          defaultValue={student.phoneNumber || undefined}
+          description="Enter your 10-digit phone number below (no formatting characters e.g. '-' or '(')."
+          error={errors.phoneNumber}
+          label="Phone Number"
+          name={keys.phoneNumber}
+          placeholder="1234567890"
```

Comment:
nit: let's change the placeholder to "5551234567", since this is more recognizable as a sample phone number.